s3_website.yml

```diff
@@ -43,7 +43,6 @@ cloudfront_distribution_config:
 # concurrency_level: 5
 
 redirects:
-  articles/can-multi-year-ssl-certificate: /articles/can-multi-year-ssl-certificates/
   articles/secondary-dns-support: /articles/secondary-dns/
```

Comment:
On second glance, it looks like you may have identified an issue with how these redirects are working. Both of these pages should redirect, but they're not right now.